card/mod/01_core/spec/set/all/references_spec.rb

```diff
@@ -3,6 +3,6 @@
 describe Card::Set::All::References do
   it "should replace references should work on inclusions inside links" do
     card = Card.create!(name: "ref test", content: "[[test_card|test{{test}}]]"  )
-    assert_equal "[[test_card|test{{best}}]]", card.replace_references("test", "best" )
+    assert_equal "[[test_card|test{{best}}]]", card.replace_reference_syntax("test", "best" )
```

Comment:
Line is too long. [93/80]<br>Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.